shoop/admin/static_src/base/js/language-changer.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * This file is part of Shoop.
+ *
+ * Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+ *
+ * This source code is licensed under the AGPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+function changeLanguage() {
+    const form = document.createElement("form");
+    form.method = "POST";
+    form.action = "/set-language/";
```

Comment:
Good catch. @shawnadelic: Add an endpoint for language changing in the admin too :)